docs/plugins/acousticbrainz.rst

```diff
@@ -14,9 +14,34 @@ Enable the ``acousticbrainz`` plugin in your configuration (see :ref:`using-plug
 For all tracks with a MusicBrainz recording ID, the plugin currently sets
 these fields:
 
-* ``danceable``: Predicts how easy the track is to dance to.
-* ``mood_happy``: Predicts the probability this track will evoke happiness.
-* ``mood_party``: Predicts the probability this track should be played at a
-  party.
+* ``danceable``
+* ``gender``
+* ``genre_rosamerica``
+* ``mood_acoustic``
+* ``mood_aggressive``
+* ``mood_electronic``
+* ``mood_happy``
+* ``mood_party``
+* ``mood_relaxed``
+* ``mood_sad``
+* ``rhythm``
+* ``tonal``
+* ``voice_instrumental``
 
-These three fields are all numbers between 0.0 and 1.0.
+Automatic Tagging
+-----------------
+
+To automatically tag files using AcousticBrainz data during import, just
+enable the ``acousticbrainz`` plugin (see :ref:`using-plugins`). When importing 
+new files (with ``import.write`` turned on) or modifying files' tags with the 
+``beet modify`` command, beets will query the AcousticBrainz API using MBID and
+set the appropriate metadata.
```

Comment:
This isn't quite right—this plugin will be triggered during import (regardless of `import.write`), and not at any other time (not during `beet modify`). Perhaps a copy-n-paste error?